LoopBackTests/LBUserTests.m

```diff
@@ -81,7 +81,7 @@ - (void)setUp {
     NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
     [defaults removeObjectForKey:DEFAULTS_CURRENT_USER_ID_KEY];
 
-    self.adapter = [LBRESTAdapter adapterWithURL:[NSURL URLWithString:@"http://localhost:3000"]];
+    self.adapter = [LBRESTAdapter adapterWithURL:[NSURL URLWithString:@"http://localhost:3000/api"]];
```

Comment:
The test servers are located under `LoopbackTests/servser` and `SLRemotingTests/servser`.
You can run the unit tests by `npm test` :-)